src/cli.rs

```diff
@@ -1175,6 +1184,31 @@ impl FromStr for Proxy {
     }
 }
 
+#[derive(Debug, Clone)]
+pub struct Resolve {
+    pub domain: String,
+    pub addr: IpAddr,
+}
+
+impl FromStr for Resolve {
+    type Err = anyhow::Error;
+
+    fn from_str(s: &str) -> anyhow::Result<Self> {
+        let (domain, addr) = s
+            .split_once(':')
+            .context("The value passed to --resolve should be formatted as <HOST>:<ADDRESS>")?;
```

Comment:
```suggestion
            .context("Value should be formatted as <HOST>:<ADDRESS>")?;
```
From
`error: invalid value 'example.com1.1.1.1' for '--resolve <HOST:ADDRESS>': The value passed to --resolve should be formatted as <HOST>:<ADDRESS>`
To
`error: invalid value 'example.com1.1.1.1' for '--resolve <HOST:ADDRESS>': Value should be formatted as <HOST>:<ADDRESS>`